<ui version="4.0" >
 <class>PreferencesClass</class>
 <widget class="QDialog" name="PreferencesClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="lblInChan" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Input Channel</string>
     </property>
     <property name="buddy" >
      <cstring>spinInChan</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QSpinBox" name="spinInChan" >
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QLabel" name="lblOutChan" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Output Channel</string>
     </property>
     <property name="buddy" >
      <cstring>spinOutChan</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QSpinBox" name="spinOutChan" >
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QLabel" name="lblVelocity" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Velocity</string>
     </property>
     <property name="buddy" >
      <cstring>spinVelocity</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QSpinBox" name="spinVelocity" >
     <property name="maximum" >
      <number>127</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QLabel" name="lblBaseOctave" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Base octave</string>
     </property>
     <property name="buddy" >
      <cstring>spinBaseOctave</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="3" >
    <widget class="QSpinBox" name="spinBaseOctave" >
     <property name="minimum" >
      <number>0</number>
     </property>
     <property name="maximum" >
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QLabel" name="lblNumOctaves" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Number of octaves</string>
     </property>
     <property name="buddy" >
      <cstring>spinNumOctaves</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="3" >
    <widget class="QSpinBox" name="spinNumOctaves" >
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="lblFileInstrument" >
     <property name="text" >
      <string>Instruments file</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QLineEdit" name="txtFileInstrument" >
     <property name="text" >
      <string>default</string>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2" >
    <widget class="QPushButton" name="btnInstrument" >
     <property name="text" >
      <string>load</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="lblInstrument" >
     <property name="text" >
      <string>Instrument</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3" >
    <widget class="QComboBox" name="cboInstrument" />
   </item>
   <item row="7" column="0" colspan="4" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>spinInChan</tabstop>
  <tabstop>spinOutChan</tabstop>
  <tabstop>spinVelocity</tabstop>
  <tabstop>spinBaseOctave</tabstop>
  <tabstop>spinNumOctaves</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>228</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel" >
     <x>263</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>118</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel" >
     <x>155</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
